lib/sorbet-schema/hash_transformer.rb

```diff
@@ -19,4 +26,30 @@ def deep_stringify_keys(hash)
       result[key.to_s] = value.is_a?(Hash) ? deep_stringify_keys(value) : value
     end
   end
+
+  private
+
+  sig { params(value: T.untyped).returns(T.untyped)}
+  def unpack_value(value)
```

Comment:
thought: Good first swing at a name, lol; I actually like it. Unfortunately, unpacking is already an overloaded term (usually deals with bit manipulation) and I don't know what box we're unpacking from here.

We're really doing a transform on the value potentially. I don't like that name either, but it might be servicable for now: `transform_value`

Reply:
we'll go with transform_value!